bika/lims/content/analysiscategory.py

```diff
@@ -38,19 +38,10 @@ def sortable_title_with_sort_key(instance):
                             "Category section on results reports."),
             label = _("Comments")),
     ),
-    ReferenceField('Department',
-        required=1,
-        vocabulary='getDepartments',
-        vocabulary_display_path_bound=sys.maxsize,
-        allowed_types=('Department',),
-        relationship='AnalysisCategoryDepartment',
-        referenceClass=HoldingReference,
-        widget=ReferenceWidget(
-            checkbox_bound=0,
-            label = _("Department"),
-            description = _("The laboratory department"),
-        ),
-    ),
+    # Department Field of Analysis Category was removed.
+    # It was a required field, but not used anywhere. When creating
+    # an Analysis Request, we use Departments from 'Analysis Services'
+    # instead.
```

Comment:
False, note the next field defined in this Schema is "DepartmentTitle", that uses the following expression:
```python
context.getDepartment() and context.getDepartment().Title() or ''
```
Are you sure neither `Department` schema's field nor `DepartmentTitle` are used anywhere?

Reply:
Yeah you are right, I didn't pay attention on `DepartmentTitle` field.
I checked now, it is not used anywhere but categories listing view. Since we are removing Departments from Categories, I think we have to remove `DepartmentTitle` column from Categories Listing View